tests/filter_fdb_input/test_vectors.py

```diff
@@ -271,8 +362,71 @@
         "arp": "tests/filter_fdb_input/arp.json",
         "fdb": "tests/filter_fdb_input/fdb.json",
         "config_db": {
+            "DEVICE_METADATA": {
+                "localhost": {
+                    "mac": "f4:8e:38:16:bc:8d" 
+                }
+            } 
         },
         "expected_fdb": [
         ],
     },
+    {
+        "arp":[
+            {
+                "NEIGH_TABLE:Vlan1000:192.168.0.10": {
+                    "neigh": "72:06:00:01:00:08",
+                    "family": "IPv4"
+                },
+                "OP": "SET"
+            },
+            {
+                "NEIGH_TABLE:Vlan1000:192.168.0.10": {
```

Comment:
is this valid scenario, having one neighbor entry map to two different macs?

Reply:
perhaps, existing code already filter out such mac, i remember previously we did correlate the fdb and arp entries, and will insert fdb entries if they exist. However, in this case, we may have fdb entry of switch mac, but I highly doubt we will also have an arp entry that points to the switch mac. Therefore, the current code should be ok.

meanwhile, I think it is good to have the check, but I think we should assign a different neighbor IP here. I do not think we can have same IP assigned to two different macs.

Reply:
Agree with Guohan.. the current code already filters out if the arp is not present and there is no valid scenario we get an arp entry (for nbr) with switch mac.

Reply:
I invented it as I did not see the arp from the switch. I'll change the IPs. The reason to have this test case is to ensure that condition `mac != switch_mac` is hit. Will update the IP address to a different one